ElixrTalentConnect/SignUP/SignupViewcontroller+Extn.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SignupViewcontroller+Extn.swift
+//  ElixrTalentConnect
+//
+//  Created by Devasurya on 15/01/24.
+//
+
+import Foundation
+import UIKit
+// Extending the SiginViewcontroller to include Tableview methods.
+extension SignUpViewController:UITableViewDelegate,UITableViewDataSource{
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return cellTypes.count
```

Comment:
I see one cellType defined in the controller, and one in the viewModel which one we are intended to use here?